Ryu64.MIPS/Registers.cs

```diff
@@ -49,5 +49,10 @@ public class COP0
 
             public static ulong[] Reg = new ulong[32];
         }
+
+        public class COP1
+        {
+            public static double[] Reg = new double[32];
```

Comment:
Wouldn't it make much more sense to make this a float?  Isn't that what the COP1 processor does?

Reply:
COP1 can do 64-bit floating point operations
![image](https://user-images.githubusercontent.com/39006831/45594559-ef60b600-b9d7-11e8-8d0e-4068f119f827.png)

Reply:
Ah okay, yes, then this looks good.